docs/md/023-VerseDonneesRole.md

```
@@ -12,9 +12,9 @@ L'utilisateur commence par naviguer à la [page de versement des données de rô
 ## Téléchargement des données
 Les données du rôle foncier peuvent être obtenues sur le site de [Données Québec](https://www.donneesquebec.ca/recherche/dataset/roles-d-evaluation-fonciere-du-quebec). Il est recommandé d'assurer la concordance des années entre le rôle foncier et le cadastre.
 ## Extraction de la municipalité pertinente et conversion geojson
-Il est suggéré de télécharger le format FGDB et d'utiliser le [script fourni](../../serveur_calcul_python/utilitaires/0_conversion_FGDB_initiale.py) pour extraire seulement les données de la municipalité concernée. Commencez par installer un environnemetn conda avec les bonnes librairies:
+Il est suggéré de télécharger le format FGDB et d'utiliser le [script fourni](../../serveur_calcul_python/utilitaires/0_conversion_FGDB_initiale.py) pour extraire seulement les données de la municipalité concernée. Commencez par installer un environnement conda avec les bonnes librairies:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)